android/src/org/mozilla/firefox/vpn/qt/VPNAppReview.java

```diff
@@ -0,0 +1,40 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.firefox.vpn.qt;
+
+import android.app.Activity;
+import android.content.Context;
+import android.util.Log;
+
+import com.google.android.play.core.review.ReviewInfo;
+import com.google.android.play.core.review.ReviewManager;
+import com.google.android.play.core.review.ReviewManagerFactory;
+import com.google.android.play.core.tasks.Task;
+
+public class VPNAppReview {
+  private static final String TAG = "VPNAppReview";
+
+  public static void appReviewRequested(Activity activity, Context context) {
```

Comment:
Does this specifically require the AppContext? 
Activity inherits from context so if not, you can just use this in `ReviewManagerFactory.create` :)